.github/actions/do_build_dpcpp/action.yml

```diff
@@ -97,6 +110,13 @@ runs:
         libclc-remangler llvm-as llvm-min-tblgen llvm-tblgen not
         opt prepare_builtins -j8
 
+    - name: cleanup ccache
+      if: contains(inputs.download_dpcpp_artefact, inputs.target) != true
```

Comment:
`!= true` looks funny, is that needed because of GitHub restrictions on expressions or could we just do `if: !contains(...)` ?